.circleci/config.yml

```diff
@@ -24,6 +24,9 @@ jobs:
           name: Set up virtual environment
           command: |
             pipenv sync --dev
+
+      - run: pipenv run ruff check .
+      - run: pipenv run ruff format .
```

Comment:
When we run `ruff format` in CI, we need to run `ruff format . --check` so it will exit with a non-zero status when there are errors

If you just run `ruff format`, when there are formatting errors it will write fixes to the files and exit with code 0

Reply:
Thanks. I actually realised this was a good chance to add a `scripts` directory. It might not end up being the final format, but at least we're moving more towards a standard than not